app/src/androidTest/java/TestPlan.kt

```diff
@@ -21,72 +19,56 @@ class TestPlan : BaseTestPlan() {
 
     @Test
     fun checkNoteScreenHeadingEditText() {
-        clickBottomToolbarButton(HOME_PAGE_BOTTOM_BAR_ADD_NOTE_BUTTON_NUMBER)
-        checkHintOnEditInRecyclerView(
-            NOTE_SCREEN_EDIT_TEXT_POSITION_FIRST,
-            NOTE_SCREEN_EDIT_TEXT_HINT_ELEMENT_THIRD,
-            getResourceString(R.string.format_hint_heading)
-        )
+        addNoteScreen.run {
+            clickAddNoteButton()
+            verifyFirstSpecificInputIsAndHasHeadingInput()
+        }
     }
 
     @Test
     fun checkNoteScreenAddContentEditText() {
-        clickBottomToolbarButton(HOME_PAGE_BOTTOM_BAR_ADD_NOTE_BUTTON_NUMBER)
-        checkHintOnEditInRecyclerView(
-            NOTE_SCREEN_EDIT_TEXT_POSITION_SECOND,
-            NOTE_SCREEN_EDIT_TEXT_HINT_ELEMENT_THIRD,
-            getResourceString(R.string.format_hint_text)
-        )
+        addNoteScreen.run {
+            clickAddNoteButton()
+            verifySecondSpecificInputIsAndHasTextInput()
+        }
     }
 
     @Test
     fun checkNoteScreenAddItemEditText() {
-        clickBottomToolbarButton(HOME_PAGE_BOTTOM_BAR_ADD_NOTE_BUTTON_NUMBER)
-        clickBottomToolbarButton(NOTE_SCREEN_BOTTOM_BAR_ADD_ITEM_BUTTON_NUMBER)
-        checkHintOnEditInRecyclerView(
-            NOTE_SCREEN_EDIT_TEXT_POSITION_SECOND,
-            NOTE_SCREEN_EDIT_TEXT_HINT_ELEMENT_FOURTH,
-            getResourceString(R.string.format_hint_list)
-        )
+        addNoteScreen.run {
+            clickAddNoteButton()
+            clickThirdButtonOnBottomToolbar()
```

Comment:
Toolbar is on the top of screen. Bottom one is BottomBar or NavigationBar - just naming.